src/controllers/invitation/updateInvitation.controller.js

```diff
@@ -0,0 +1,47 @@
+import { Invitation, Project, Organization } from "../../config/db.js"
+
+const updateInvitationController = async (req, res) => {
+  const { id, state, accessType } = req.body
+  if (!state || !accessType || !id) return res.status(400).send("You must send all required info")
+
```

Comment:
i would check that state value is a permitted value
```suggestion
if (! ["Accepted", "Declined", ...].includes(state)) return res.status(400).send(`${state} is not valid invitation state`)
```